### PR TITLE
conn: udp: doc fix

### PR DESCRIPTION
**Pull request by cgundogan**

### CODE REVIEW

File: sys/include/net/conn/udp.h [hunk 1]
```diff
@@ -105,9 +105,9 @@ int conn_udp_recvfrom(conn_udp_t *conn, void *data, size_t max_len, void *addr,
 /**
  * @brief   Sends a UDP message
  *
- * @param[in] data      Pointer where the received data should be stored.
- * @param[in] len       Maximum space available at @p data.
- * @param[in] src       The source address. May be NULL for all any interface address.
+ * @param[in] data      Pointer to the payload.
```
Comment (authmillenon) [inline sys/include/net/conn/udp.h]: What's "the payload" in this context? I'm asking from a perspective of a person that does not know much about UDP.

Comment (cgundogan) [inline sys/include/net/conn/udp.h]: Data that should be put into the packet. The term payload is layer agnostic so it shouldn't be a problem that someone does not understand UDP. Payload can be plain user data or a home-brewed coap header. I see no need to explain `payload` further in this context.

Comment (authmillenon) [inline sys/include/net/conn/udp.h]: But why is the current doc then insufficient?

Comment (cgundogan) [inline sys/include/net/conn/udp.h]: the current doc talks about the **pointere** where the **received** data should be **stored**.
AFAIK, `conn_udp_sendto` should send and not receive?

Comment (authmillenon) [inline sys/include/net/conn/udp.h]: This is indeed correct. But to keep symmetry with `recvfrom` I would still use the same terminology.

Comment (cgundogan) [inline sys/include/net/conn/udp.h]: `Pointer from where the payload is read from`?

Comment (authmillenon) [inline sys/include/net/conn/udp.h]: `Data to send` ;-)

**Comment by cgundogan** [comment]
@authmillenon addressed your comment and squashed

**Comment by authmillenon** [comment]
ACK and go, when Travis is happy.

**Comment by cgundogan** [comment]
travis is okay with the change. The fail is due to relic not being able to build (fixed in #4743).

**Pull request merged**